src/app/user-page/user-page.component.html

```diff
@@ -0,0 +1,132 @@
+<app-header>
+  <div fxLayout fxLayoutAlign="space-between stretch">
+    <div fxHide.lt-sm fxLayout fxLayoutGap="1rem">
+      <span><mat-icon>people</mat-icon></span>
+      <div>{{ user?.username }}</div>
+    </div>
+  </div>
+</app-header>
+
+<div class="container">
+  <div fxLayout fxLayout.lt-md="column" fxLayoutAlign="center" fxLayoutGap="1rem" class="my-5">
+    <mat-card fxFlex.gt-md="25" fxFlex.lt-md="100" fxLayout="column" fxLayoutAlign="center">
+      <img mat-card-image [src]="user?.avatarUrl" class="mt-0 mx-auto mb-3" />
```

Comment:
If I read this correctly, until the `user` is loaded, `src` will be undefined, and while so, the `img` tag might display a "broken image" icon on some browsers.  To see if that's true, you can insert a delay into the code that retrieves `user` from the server, and see what happens.  If it looks bad, maybe add an `*ngIf="user"` to the `mat-card` , or something like that?

Reply:
mm for `[src]="user?.avatarUrl"` there is the ? operator which should prevent the broken picture since it should check first if `user` is null/undefined, and if so, it does not display an `img` at all so I think that should be okay?

But that does remind me of an issue with google images that I'll comment on this PR

Reply:
Angular evaluates the expression `user?.avatarUrl`, which will be `undefined` initially, and then sets the `src` attribute to the resulting value.  I wasn't sure if that meant Angular would not include the attribute, or include the attribute and leave it unset, set it to a string like "" or "undefined", or something else.  I ran a quick experiment, and it looks like Angular is setting the `src` attribute to `undefined`, which makes the `img` tag simply not appear in the browsers I checked.  So yes, your code is probably fine.  The caveat being that the `src` attribute is required to be a valid URL by the HTML spec (https://html.spec.whatwg.org/#attr-img-src), so all browsers may not behave as gracefully...

Reply:
okay adding the extra ngif on the card to be safe!